lambda/health_package/components/cloudwatch_helper.py

```diff
@@ -0,0 +1,102 @@
+""" Cloudwatch component helper functions """
+import json
+
+import botocore
+from addict import Dict
+
+from components.generic_helper import GenericHelper
+# from logger import LOG
+
+
+class CloudwatchHelper(GenericHelper):
+    """ Helper functions for Cloudwatch """
+
+    @classmethod
+    def metric_resource_exists(cls, metric, region=None):
+        """
+        Check the resource exists before defining an alarm
+        aws cloudwatch list-metrics returns metrics for resources that
+        no longer exists
+        """
+        namespace = metric.Namespace
+        resource_exists = True
+        try:
+            print(f"Getting boto client for {namespace} in {region}")
+            client = cls.get_client_from_namespace(namespace, region)
+            if client:
+                event_rule_name = cls.get_metric_dimension_value(metric, "RuleName")
+                if event_rule_name:
+                    print(f"Get tags for event rule names: {event_rule_name}")
+                    client.list_targets_by_rule(Rule=event_rule_name)
+                else:
+                    resource_exists = False
+
+        except AttributeError as err:
+            print(json.dumps(metric, indent=2))
+            print(str(err))
+        except botocore.exceptions.ClientError as err:
+            print(str(err))
+            resource_exists = False
+        return resource_exists
+
+    @classmethod
+    def get_tags_for_metric_resource(cls, metric, region=None):
```

Comment:
Can you add a simple doc string here, please?

Reply:
Sure will add one.